src/ScrcpyNet/VideoStreamDecoder.cs

```diff
@@ -26,7 +26,7 @@ public ReadOnlySpan<byte> Data
 
         public int Width { get; }
         public int Height { get; }
-        public int FrameNumber { get; }
+        //public int FrameNumber { get; }
```

Comment:
Please either delete or provide a reason why this code is left commented.

Reply:
My fault. After upgrading FFmpeg.AutoGen, the AVCodecContext.frame_number(int) was refactored with AVCodecContext.frame_num(long)